beamerInitialize/index.js

```diff
@@ -6,7 +6,8 @@ Object.defineProperty(exports, "__esModule", {
 exports.default = beamerInitialize;
 
 function beamerInitialize(id) {
-  var url = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : false;
+  var url = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : null;
```

Comment:
let's change here to actually have the params instead of using arguments.

```
function beamerInitialize(id, url, config) {
[...]

window.beamer_config = {
 product_id: id,
 ...(config || {}),
}

[...]
```

this should clean up the var definition.